spectrocloud/cluster_common_hash.go

```diff
@@ -268,7 +268,7 @@ func resourceMachinePoolMaasHash(v interface{}) int {
 		}
 	}
 	buf.WriteString(fmt.Sprintf("%s-", m["azs"].(*schema.Set).GoString()))
-
+	buf.WriteString(fmt.Sprintf("%s-", m["node_tags"].(*schema.Set).GoString()))
```

Comment:
@SivaanandM unit test is failing, could you please check across the code use case where node_tags are nil?

Reply:
done